src/Core/src/Platform/Android/NativeVersion.cs

```diff
@@ -29,5 +29,6 @@ public static class NativeApis
 		public const int PowerSaveMode = 21;
 		public const int BlendModeColorFilter = 29;
 		public const int SeekBarSetMin = 26;
+		public const int StatusBarColor = 21;
```

Comment:
Since we only support 21+ do we need this?

Reply:
If not, `PowerSaveMode` also needs to be removed.